metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "puppetlabs-rabbitmq",
+  "name": "puppet-rabbitmq",
   "version": "5.6.0",
```

Comment:
Maybe update version to an 'rc' version? 6.0.0-rc??

Reply:
works for me, 6.0.0-rc0? I figured that might get set when we do modulesync, but I don't see any reason not to update it, and next release will have a lot of breaking changes, I think.